ci-operator/step-registry/openshift/e2e/aws/csi/efs/sts/extended/openshift-e2e-aws-csi-efs-sts-extended-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-aws-csi-efs-sts-extended
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service-provision
+    - chain: storage-conf-csi-optional-aws-efs-sts
+    test:
+    - ref: openshift-extended-test
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service-deprovision
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FIPS_ENABLED: "true"
```

Comment:
Is FIPS really enabled in the cluster? It would be great if it was, and the workflow + CI job should have `-fips-sts` in the name.

Reply:
Yeah, fips is enabled in the cluster. Good point, let me update the workflow name to make it more clearly.